inc/vcf/assembly_report_writer.hpp

```diff
@@ -87,29 +142,33 @@ namespace ebi
 
         ~TextAssemblyReportWriter()
         {
+            file << "Number of matches: " << match_stats.get_num_matches()
+                 << "/" << match_stats.get_num_variants() << std::endl;
+            file << "Percentage of matches: "
+                 << (static_cast<double>(match_stats.get_num_matches()) / match_stats.get_num_variants()) * 100
+                 << "%" << std::endl;
             file.close();
         }
 
-        virtual void write_mismatch(const vcf::RecordCore &record_core) override
+        virtual void match(const vcf::RecordCore &record_core, std::string &match_result) override
+        {
+            match_stats.add_match_result(true);
```

Comment:
Same comment as in `ValidAssemblyReportWriter`, only the summary should need the statistics.

Reply:
done that using a local bool variable `is_valid`